mock-core-configs/etc/mock/templates/oraclelinux-7.tpl

```diff
@@ -0,0 +1,56 @@
+# This list is taken from 'epel-7-x86_64' @buildsys-build group, minus the
+# 'epel-*' specific stuff.
+config_opts['chroot_setup_cmd'] = 'install bash bzip2 coreutils cpio diffutils findutils gawk gcc gcc-c++ grep gzip info make patch redhat-rpm-config rpm-build sed shadow-utils tar unzip util-linux which xz'
+
+config_opts['dist'] = 'el7'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '7'
+config_opts['bootstrap_image'] = 'docker.io/library/oraclelinux:7'
+config_opts['package_manager'] = 'yum'
+
+config_opts['yum_install_command'] += "{% if target_arch in ['x86_64', 'ppc64le', 'aarch64'] %} --disablerepo=ol7_software_collections{% endif %}"
+
+config_opts['yum.conf'] = """
+[main]
+keepcache=1
+debuglevel=2
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=mock
+syslog_device=
+metadata_expire=0
+mdpolicy=group:primary
+best=1
+protected_packages=
+user_agent={{ user_agent }}
+
+# repos
+[ol7_latest]
+name=Oracle Linux $releasever Latest ($basearch)
+baseurl=https://yum.oracle.com/repo/OracleLinux/OL7/latest/$basearch/
+gpgkey=file:///usr/share/distribution-gpg-keys/oraclelinux/RPM-GPG-KEY-oracle-ol7
+gpgcheck=1
+enabled=1
+skip_if_unavailable=False
+
+[ol7_optional_latest]
+name=Oracle Linux $releasever Optional Latest ($basearch)
+baseurl=https://yum.oracle.com/repo/OracleLinux/OL7/optional/latest/$basearch/
+gpgkey=file:///usr/share/distribution-gpg-keys/oraclelinux/RPM-GPG-KEY-oracle-ol7
+gpgcheck=1
+enabled=1
+skip_if_unavailable=False
+
+[ol7_software_collections]
+name=Software Collection Library packages for Oracle Linux 7 ($basearch)
+baseurl=https://yum.oracle.com/repo/OracleLinux/OL7/SoftwareCollections/$basearch/
+gpgkey=file:///usr/share/distribution-gpg-keys/oraclelinux/RPM-GPG-KEY-oracle-ol7
+gpgcheck=1
+enabled=1
+includepkgs=devtoolset*
+skip_if_unavailable=False
```

Comment:
This should probably be `skip_if_unavailable=True`

Reply:
Well, we want it to fail if packages aren't fetchable, I think?

Reply:
If `devtoolset*` is only for EPEL layering, I'm more inclined to suggest this entire repo be added to a future `oraclelinux7-epel.tpl` but I also can't come up with a scenario in which `ol7_latest` is available and `ol7_software_collection` is not, so 🤷🏻

Reply:
We can always change this later if we want to come up with a fancier abstraction. I just wanted to mimic the CentOS one closely for fungibility.

Reply:
Fair enough.